utils.py

```diff
@@ -405,13 +412,23 @@ def get_time_in_millisecs(datetime_obj):
 
 
 def get_current_time_in_millisecs():
-    """Returns time in milliseconds since the Epoch."""
+    """Returns time in milliseconds since the Epoch.
+    
+    Return:
+        float. Returns time in millisecond.
+    """
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
 def get_human_readable_time_string(time_msec):
     """Given a time in milliseconds since the epoch, get a human-readable
     time string for the admin dashboard.
+    
+    Arg:
+        time_msec: double. Passing time in milliseconds.
+    
+    Return:
+        float. Time in human redable format.
```

Comment:
Can you please recheck the return type? I don't think it should be a float.

Reply:
done